ReactWindows/js/Components/Picker/PickerWindows.js

```diff
@@ -0,0 +1,137 @@
+ */
+var PickerWindows = React.createClass({
+
+  propTypes: {
+    ...View.propTypes,
+    style: pickerStyleType,
+    items: React.PropTypes.any,
+    selected: React.PropTypes.number,
+    selectedValue: React.PropTypes.any,
+    enabled: ReactPropTypes.bool,
+    onValueChange: ReactPropTypes.func,
+    prompt: ReactPropTypes.string,
+    testID: ReactPropTypes.string,
+  },
+
+  getInitialState: function() {
+    return this._stateFromProps(this.props);
+  },
+
+  componentWillReceiveProps: function(nextProps) {
+    this.setState(this._stateFromProps(nextProps));
+  },
+
+  // Translate prop and children into stuff that the native picker understands.
+  _stateFromProps: function(props) {
+    var selectedIndex = 0;
+    let items = ReactChildren.map(props.children, (child, index) => {
+      if (child.props.value === props.selectedValue) {
+        selectedIndex = index;
+      }
+      let childProps = {
+        value: child.props.value,
+        label: child.props.label,
+      };
+      if (child.props.color) {
+        childProps.color = processColor(child.props.color);
+      }
+      return childProps;
+    });
+    return {selectedIndex, items};
+  },
+  
+  render: function() {
+    var Picker = ComboBoxPicker;
+
+    var nativeProps = {
+      enabled: this.props.enabled,
+      items: this.state.items,
+      onSelect: this._onChange,
+      prompt: this.props.prompt,
+      selected: this.state.selectedIndex,
+      testID: this.props.testID,
+      style: [styles.pickerWindows, this.props.style],
+    };
+
+    return <Picker ref={REF_PICKER} {...nativeProps} />;
+  },
+
+  _onChange: function(event: Object) {
+    if (this.props.onValueChange) {
+      var position = event.nativeEvent.position;
+      if (position >= 0) {
+        var value = this.props.children[position].props.value;
+        this.props.onValueChange(value, position);
+      } else {
+        this.props.onValueChange(null, position);
+      }
+    }
+
+    // The picker is a controlled component. This means we expect the
+    // on*Change handlers to be in charge of updating our
+    // `selectedValue` prop. That way they can also
+    // disallow/undo/mutate the selection of certain values. In other
+    // words, the embedder of this component should be the source of
+    // truth, not the native component.
+    if (this.refs[REF_PICKER] && this.state.selectedIndex !== event.nativeEvent.position) {
+      this.refs[REF_PICKER].setNativeProps({selected: this.state.selectedIndex});
+    }
+  },
+});
+
+var styles = StyleSheet.create({
+  pickerWindows: {
+    // The picker will conform to whatever width is given, but we do
+    // have to set the component's height explicitly on the
+    // surrounding view to ensure it gets rendered.
+    // TODO would be better to export a native constant for this,
+    // like in iOS the RCTDatePickerManager.m
+    height: 40,
+  },
+});
+
+var cfg = {
+  nativeOnly: {
+  }
```

Comment:
I added them (selected / items) back to be the same as Android code - even though I don't think they are needed, since PickerWindows.js is not exposed as an API.